Source/SwiftLintFramework/Rules/FunctionParameterCountRule.swift

```diff
@@ -91,6 +101,14 @@ public struct FunctionParameterCountRule: ASTRule, ConfigurationProviderRule {
             .characters.filter { $0 == equalCharacter }.count ?? 0
     }
 
+    fileprivate func functionIsInitializer(_ file: File, offset: Int, length: Int) -> Bool {
+        if let function = (file.contents as NSString)
```

Comment:
If I understood correctly, @norio-nomura was explicit in his comment and suggested using `NSString`. If you search for `(file.contents as NSString)` in the project you'll see that sometimes this conversion is used even if redundant. But I can be wrong here.